website/docs/integrations/trello.mdx

```diff
@@ -4,23 +4,67 @@ title: Trello Power-Up - Manage feature flags from Trello
 description: ConfigCat Trello Power-Up. This is a step-by-step guide on how to connect and manage feature flags from Trello boards and cards.
 ---
 
-Turn features On / Off right from a linked Card on your Trello board with <a href="https://trello.com/power-ups/5e694b66d2511a3601ebd0fb" target="_blank">ConfigCat's Trello Power-Up</a>. Also you can add Targeting or Percentage Rules just as easily.
+The <a href="https://trello.com/power-ups/5e694b66d2511a3601ebd0fb" target="_blank">ConfigCat's Trello Power-Up</a> allows you to connect your Trello Cards and feature flags. Create or link existing flags to your Trello cards without leaving your Trello board.
 
-## Installation
+Turn features On/Off right from a linked card on your Trello board. You can also easily modify the linked flags to edit or add new Targeting or Percentage Rules.
+
+This guide will help you with the power-up installation and familiarise you with the plugin usage.
 
-<img src="/docs/assets/trello_enable.gif" className="zoomable" alt="Installation of the ConfigCat Feature Flags Trello Power-Up" />
+## Installation
 
 1. Add <a href="https://trello.com/power-ups/5e694b66d2511a3601ebd0fb" target="_blank">ConfigCat Power-Up</a> to your board.
 2. Select `Settings -> Authorize Account`.
-3. Get your ConfigCat Public API credentials: https://app.configcat.com/my-account/public-api-credentials
+3. Copy your ConfigCat Public API credentials to the inputs. Read more about <a href="https://app.configcat.com/my-account/public-api-credentials" target="_blank">ConfigCat Public API credentials here</a>.
 4. Click authorize.
 
+:::info
+Every Trello user must authorize ConfigCat in Trello who wants to use the ConfigCat Feature Flags power-up.
+:::
+
+<img src="/docs/assets/trello/install.gif" className="zoomable" alt="Installation of the ConfigCat Feature Flags Trello Power-Up" />
+
 ## Usage
 
-<img src="/docs/assets/trello_use.gif" className="zoomable" alt="Usage of the ConfigCat Feature Flags Trello Power-Up" />
+### Creating new feature flags
 
 1. Open any Card on your Trello board.
-2. Push `Link Feature Flag`
```

Comment:
Ebben az esetben is az igaz, hogy a userek sokkal gyakrabban linkelnek, mint creatolnak, ezért a legjobb az lenne, ha a doksiban és a plugin GUI-n is a Link előbb lenne, mint a Create